File: src/client_gen/fsharp/MyStrategy.fs
namespace ProjectName

open ProjectName.Model

type MyStrategy() =
    member this.getAction(playerView: PlayerView, debugInterface: DebugInterface): Action =
        raise (System.NotImplementedException "Write your strategy here")

    member this.debugUpdate(playerView: PlayerView, debugInterface: DebugInterface) = ()

File: src/client_gen/go/my_strategy.go
package main

import (
	. "project_name/model"
)

type MyStrategy struct{}

func NewMyStrategy() MyStrategy {
	return MyStrategy{}
}

func (strategy MyStrategy) getAction(playerView PlayerView, debugInterface *DebugInterface) Action {
	panic("Write your strategy here")
}

func (strategy MyStrategy) debugUpdate(playerView PlayerView, debugInterface DebugInterface) {
	debugInterface.Send(DebugCommandClear{})
	debugInterface.GetState()
}
